microcosm_postgres/store.py

```diff
@@ -34,9 +34,13 @@
 
 class Store:
 
-    def __init__(self, graph, model_class):
+    def __init__(self, graph, model_class, auto_filter_fields=()):
         self.graph = graph
         self.model_class = model_class
+        self.auto_filters = {
+            auto_filter_field.name: auto_filter_field
+            for auto_filter_field in auto_filter_fields
+        }
```

Comment:
Is it too risky to just autofilter every column in the model by default?

Reply:
Not just risky... not desirable.

We want filtering (and indexing) to be explicit choices. Not magic.